### DISCUSSION TITLE
OS X El Capitan 10.11.6

### DISCUSSION THREAD

**Discussion by dmtzu** [lmparppei/Beat#124] [Q&A]
Hello. Where can I download the version for OS X El Capitan 10.11.6 ?

**Comment by lmparppei** [comment]
Unfortunately Beat won't work below macOS 10.12. In theory, somebody *could* make it compatible with 10.10-11 by removing preview and plugin support, and replacing some parts of the UI.

For now, Beat is unavailable for older systems. My apologies!
